### PR TITLE
[CDAP-16571] Add empty indicator for plugin selector

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-16571
Build: https://builds.cask.co/browse/CDAP-UDUT571

### CODE REVIEW

File: cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ const CustomSelect: React.FC<ISelectProps> = ({
       }}
     >
       {optionValues.map((opt) => (
-        <MenuItem value={opt.value} key={opt.value}>
+        <MenuItem value={opt.value} key={opt.value} disabled={opt.disabled}>
```
Comment (yukiej) [inline cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx:78]: Do we need to add a default here for opt.disabled?

File: cdap-ui/app/cdap/components/AbstractWidget/PluginListWidget/index.tsx [hunk 2]
```diff
@@ -60,6 +60,19 @@ const PluginListWidget: React.FC<IPluginListProps> = ({
     };
 
     MyPipelineApi.getExtensions(params).subscribe((res: IPlugin[]) => {
+      if (res.length === 0) {
+        const emptyOptions = [
+          {
+            value: '',
+            label: `No ${params.extension} plugin`,
```
Comment (yukiej) [inline cdap-ui/app/cdap/components/AbstractWidget/PluginListWidget/index.tsx]: nit: I think it should be plural, i.e. "no {extension} plugins"

Also, i18n?

File: cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx:78]: no, if it's falsy it won't disable it

**Review by yukiej — APPROVED** [review]
LGTM!

**Pull request merged**